### ISSUE TITLE
Mnemonic shortcuts missing in many Dialogue Boxes

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#20095]
### Issue type

UI bug (incorrect info or interface appearance)

### Bug description

In most dialogue boxes mnemonic shortcuts for screen readers do not seem to be present or if they are present are not being advertised to screen readers.  It is assumed that Y for Yes, N for No, Esc for close and Enter for OK will work in any dialogue box.  But others like Alt N for Next, Alt F for Finish, etc will be also familiar and automatic to most screen readers.  These seem to be missing in most cases and in the case of sub dialogue boxes in the new score wizard the OK and Cancel are completely missing as buttons and Esc rather than enter is used to finish saving the info.

Two examples are:
•	In New Score Wizard, Alt+N should activate Next button
•	In Confirm Quit dialog, Alt+S should save and Alt+N for Don't save
•	

### Steps to reproduce

1.  Open any score and make a few changes.
2. Attempt to close the score without saving.  The Confirm save changes dialogue box appears
3. Arrow left and right to hear the options and notice no mnemonic shortcuts are announced by the screen reader.
4. Try pressing Alt S to Save.  Note - Nothing happens.
5. Try pressing Alt D for Don't save, nothing happens.

### MuseScore Version

4.1

### Regression

I don't know

### Operating system

Windows 10 with NVDA
